fe/fe-core/src/main/java/org/apache/doris/job/base/AbstractJob.java

```diff
@@ -303,7 +348,19 @@ public ShowResultSetMetaData getJobMetaData() {
         return builder.build();
     }
 
-    private static long getNextId() {
```

Comment:
Why changing the method of generating task id?

Reply:
we will generate task id in constructor